nio-impl/src/main/java/org/xnio/nio/WorkerThread.java

```diff
@@ -334,7 +334,7 @@ void handleReady(final int ops) {
                     }
                 }
             } catch (IOException e) {
-                selectorLog.tracef("ConnectHandle.handleReady Exception, " + e);
+                selectorLog.tracef("ConnectHandle.handleReady Exception, %s", e);
```

Comment:
I did not modify behavior of this statement, but we may want to log the full stack trace here